api/service_accounts_handlers.go

```diff
@@ -156,12 +157,38 @@ func serviceAccountsListHandler(
 			)
 			return
 		}
-		saSl, count, err := sasUC.WithContext(r.Context()).List(listOptions)
+
+		var saSl []models.ServiceAccount
```

Comment:
Now I get what you were saying about using the same route. I don't like this approach.

It's much cleaner to have different routes for different issues. Something like: `GET /service_accounts/with_permission?permission=`